### PR TITLE
fix: Centralize docker networks

### PR DESCRIPTION
**Pull request by seanohair22**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #**90**
closes: #90 
## Test Instructions if applicable

- Optional: Run `make run-portainer`
- Run `make run-base` or `run-full` and check the docker networks that are setup. 
- Note: Portainer runs on a separate docker network in order to spin up and take down the RTSF services easily and still use Portainer.

### CODE REVIEW

File: loss-detection-app/docker-compose.vap.yml [hunk 1]
```diff
@@ -17,7 +17,8 @@ version: '3.7'
 
 networks:
   loss-detection-app_edgex-network:
-    external: true
```
Comment (lenny-intel) [inline loss-detection-app/docker-compose.vap.yml:20]: The VAP compose isn't run when simulating, so I think the network should be defined in `loss-detection` compose like it was.

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: loss-detection-app/docker-compose.vap.yml (hunk 1 above)
Comment (seanohair22) [reply loss-detection-app/docker-compose.vap.yml]: Okay, addressed!

File: loss-detection-app/docker-compose.loss-detection.yml [hunk 2]
```diff
@@ -17,7 +17,8 @@ version: '3.7'
 
 networks:
   loss-detection-app_edgex-network:
-    external: true
+    name: loss-detection-app_edgex-network
+    driver: bridge
```
Comment (lenny-intel) [inline loss-detection-app/docker-compose.loss-detection.yml:21]: How is this working with the network defined in the edgex compose file?

**Review by lenny-intel — APPROVED** [review]
LGTM

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Pull request merged**